modules/installation-aws-permissions.adoc

```diff
@@ -7,6 +7,11 @@
 [id="installation-aws-permissions_{context}"]
 = Required AWS permissions for the IAM user
 
+[NOTE]
+====
+Your IAM user must have the permission `tag:GetResources` in the region `us-east-1` to delete the base cluster resources. As part of the AWS API requirement, the `openshift-installer` performs various actions in this region.
```

Comment:
the `openshift-installer` -> the {product-title} installation program

Reply:
Addressed